doc/us-en/FAQ.md

```diff
@@ -64,4 +64,26 @@ export DOCKER_API_VERSION=1.38
 
 **Question 7**: How does BIE access the NB-IOT network?
 
-NB-IoT is a network standard similar to 2/3/4G with low bandwidth and low power consumption. NB-IoT supports TCP-based MQTT protocol, so you can use NB-IoT card to connect to Baidu Cloud IotHub, deploy OpenEdge application and communicate with BIE Cloud Management Suite. However, among the three major operators in China, Telecom have imposed whitelist restrictions on their NB cards, and only allow to connect to Telecom Cloud Service IP. Therefore, only Mobile NB cards and Unicom NB cards can be used to connect to Baidu Cloud Service.
\ No newline at end of file
+NB-IoT is a network standard similar to 2/3/4G with low bandwidth and low power consumption. NB-IoT supports TCP-based MQTT protocol, so you can use NB-IoT card to connect to Baidu Cloud IotHub, deploy OpenEdge application and communicate with BIE Cloud Management Suite. However, among the three major operators in China, Telecom have imposed whitelist restrictions on their NB cards, and only allow to connect to Telecom Cloud Service IP. Therefore, only Mobile NB cards and Unicom NB cards can be used to connect to Baidu Cloud Service.
+
+**Question 8**: var/run/openedge.sock: address already in use
+
+Remove var/run/openedge.sock and restart Openedge.
+
+**Question 9**: Does openedge support to push data to kafka?
+
+For support, you can use the function framework to write a function that is responsible for subscribing messages from the Hub and writing them to Kafka one by one. You can also customize the module, which subscribes to the Hub and then writes kafka in bulk.
```

Comment:
For support, you can use the local function module to write a [Python script](./customize/How-to-write-a-python-script-for-python-runtime.md) that is responsible for subscribing messages from the local Hub module and writing them to Kafka service. Besides, you can also [develop a customize module](./customize/How-to-develop-a-customize-module-for-openedge.md), which subscribes message from the local Hub module and then writes it to Kafka.